Vision2022/main.py

```diff
@@ -170,23 +175,32 @@ def send(data):
     s.sendto(data.encode('utf-8'), address)
 
 
-def send_data(hoop_found, hoop_x, hoop_y, hoop_angle, ball_found, ball_x, ball_y, ball_angle, wait_for_other_robot):
+def send_data(hoop_x, hoop_y, hoop_angle, ball_found, ball_x, ball_y, ball_angle, wait_for_other_robot):
+
     data = '%3.1f %3.1f %3.1f %3.1f %3.1f %3.1f %d %d %d \n' % (
-    hoop_x, hoop_y, hoop_angle, ball_x, ball_y, ball_angle, int(ball_found), int(ball_found),
-    wait_for_other_robot)
+        hoop_x, hoop_y, hoop_angle, ball_x, ball_y, ball_angle, int(ball_found), int(ball_found),
```

Comment:
This doesn't seem right - we're returning `ball_found` twice but not `hoop_found`.